python/lsst/pipe/base/pipelineIR.py

```diff
@@ -619,10 +619,14 @@ def process_args(argument: Union[str, dict]) -> dict:
             if self.instrument is None:
                 self.instrument = tmp_IR.instrument
             elif self.instrument != tmp_IR.instrument and tmp_IR.instrument is not None:
-                raise ValueError("Only one instrument can be declared in a pipeline or it's imports")
-            if accumulate_tasks.keys() & tmp_IR.tasks.keys():
-                raise ValueError("Task labels in the imported pipelines must "
-                                 "be unique")
+                msg = ("Only one instrument can be declared in a pipeline or its imports. "
+                       f"These instruments were found: {self.instrument}, {tmp_IR.instrument}")
```

Comment:
If you are expanding the exception message, might as well also say something like:
```
"Top level pipeline defines {self.instrument} but {other_pipeline.location} defines {tmp_IR.instrument}"
```
To help people track down where the other declaration was.

Reply:
Thank you, that's a very useful addition.